persistence-api/src/main/java/io/stargate/db/AuthenticatedUser.java

```diff
@@ -16,7 +16,6 @@
 package io.stargate.db;
 
 import com.datastax.oss.driver.shaded.guava.common.collect.ImmutableMap;
-import com.datastax.oss.driver.shaded.guava.common.collect.ImmutableMap.Builder;
```

Comment:
This was done due to ErrorProne warning wrt "too generic static imports" (same for similar changes later on).